external/external_custom/playlist.xml

```diff
@@ -24,7 +24,7 @@
 			<level>extended</level>
 		</levels>
 		<groups>
-			<group>external</group>
+			<group>functional</group>
```

Comment:
why do we move the external test to the functional group?

Reply:
Typo.  Updated.